com.somaxa8.earx.yml

```diff
@@ -0,0 +1,36 @@
+app-id: com.somaxa8.earx
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: earx
+finish-args:
+  - --system-talk-name=org.bluez
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --allow=bluetooth
+
+modules:
+  - name: earx
+    build-options:
+      append-path: /usr/lib/sdk/rust-stable/bin
+      env:
+        CARGO_HOME: /run/build/earx/cargo
+    buildsystem: simple
+    build-commands:
+      - cargo --offline fetch --manifest-path Cargo.toml --verbose
+      - cargo --offline build --release --verbose
+      - install -Dm755 target/release/earx -t $FLATPAK_DEST/bin
+      - install -Dm644 assets/ear.png $FLATPAK_DEST/assets/ear.png
+      - install -Dm644 assets/logo-64x64.png $FLATPAK_DEST/share/icons/hicolor/64x64/apps/com.somaxa8.earx.png
+      - install -Dm644 assets/logo-128x128.png $FLATPAK_DEST/share/icons/hicolor/128x128/apps/com.somaxa8.earx.png
+      - install -Dm644 com.somaxa8.earx.desktop $FLATPAK_DEST/share/applications/com.somaxa8.earx.desktop
+      - install -Dm644 com.somaxa8.earx.metainfo.xml $FLATPAK_DEST/share/metainfo/com.somaxa8.earx.metainfo.xml
```

Comment:
Add https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#url https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#developer-name and https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#open-age-ratings-service-oars